scraping/scraper.py

```diff
@@ -67,16 +67,20 @@ def scrape_cluster(workspace, cluster_dict, instance_types, session, api, result
         state_message=cluster_dict["state_message"],
         driver_type=cluster_dict["driver_node_type_id"],
         worker_type=cluster_dict["node_type_id"],
-        num_workers=cluster_dict["num_workers"],
+        num_workers=cluster_dict.get("num_workers", 0),
+        autoscale_min_workers=(cluster_dict
+                               .get('autoscale', {})
+                               .get('min_workers', 0)),
```

Comment:
Shouldn't we set it to `None` to indicate, that this value is not applicable if autoscaling is not enabled?